lib/src/main/kotlin/com/santaclose/lib/entity/mountainReview/Ratings.kt

```diff
@@ -0,0 +1,10 @@
+package com.santaclose.lib.entity.mountainReview
+
+import javax.persistence.Embeddable
+
+@Embeddable
+class Ratings(
+    var scenery: Int,
```

Comment:
마스터 리베이스 하신 다음에 0이상만 가능하도록 validation 을 추가하면 좋을거 같습니다!
그리고 평점이 최대 5점이라 Int 대신 더 작은 Byte 로 가도 괜찮을거 같은데 어떻게 생각하시나요?

```suggestion
    @field:Min(0)
    var scenery: Byte,
```

Ratings 는 Embeddable 이라서 MountainReview 에 다음처럼 선언해야 합니다.

```kt
    @field:Valid
    @Embedded
    var ratings: Ratings,
```

Reply:
넵 무척 좋은 의견입니당!
감사합니다! :)

Reply:
검색해보니까 `@field:PositiveOrZero` 가 있네요. ㅎㅎ
이거쓰는게 더 좋아보여요.

Reply:
@jbl428 음.. 최대치가 있을 수 있으니까
`@field:Size(min = 0, max = 5)` 이렇게는 어떠세용?

Reply:
넵 그게 더 좋네요 ㅎㅎ